src/base/torrentfileguard.h

```diff
@@ -77,9 +77,9 @@ class TorrentFileGuard : private FileGuard
 
 private:
     TorrentFileGuard(const Path &path, AutoDeleteMode mode);
-    static SettingValue<AutoDeleteMode> &autoDeleteModeSetting();
```

Comment:
Don't understand what you mean. It still works now.

Reply:
Well, it can still be changed using returned copy.

Another question is why don't just declare class variable instead of method like we do in other similar cases?

Reply:
You must misunderstood. I don't strictly reject the use of static variables but I think their usage must be minimized and under close watch.

This code here doesn't seem like used frequently enough IMO, or if you think otherwise, I can make it a class variable.

Reply:
> It is used in static methods... I don't know why it is this way.

Ok, forget it. Another cases I mentioned above aren't static.

Reply:
>The other instance in searchjobwidgets can be changed, I'll update it later.

Done.